util/markup.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from dominate.tags import a
```

Comment:
How are you reimplementing the wheel?

     a(employee, href=url_for('explore', user=employee)).render()
    '<a href="{}">{}</a>'.format(url_for('explore', user=employee), employee)

I'm generally wary of adding these kinds of dependencies, it's just one more thing to maintain that adds bloat to our project. And it's not like a) we expect changes to how a tags are built that this package will cover for us; or b) that we not already have folders full of HTML. I consider this a general feeling of not wanting to put HTML markup in a "code" file, while the library just hides the fact that you do this.